searchcore/src/vespa/searchcore/config/ranking-expressions.def

```diff
@@ -0,0 +1,5 @@
+# Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+namespace=vespa.config.search.core
+
+constant[].name string
```

Comment:
Is 'constant' the correct name, or should it be 'expression' ?

Reply:
fixed